packages/@okta/vuepress-theme-prose/components/TopBar.vue

```diff
@@ -1,70 +1,45 @@
+              </svg>
+            </a>
+          <nav :class="{'navigation':true, 'expanded': !topBarCollapsed}">
+            <ul class="menu">
+              <li v-for="(link, index) in $themeConfig.primary_nav" :key="index" :class="{'hasSubMenu': link.children}">
+                <a :href="link.link" :class="{'active': $page.path.includes(link.link)}" >{{link.text}}</a><span v-if="link.children" class="caret">◥◤</span>
+                <ul v-if="link.children" class="submenu">
+                  <li v-for="(childLink, childIndex) in link.children" :key="childIndex">
+                    <a v-if="childLink.link" :href="childLink.link" class="link">{{childLink.text}}</a>
+                    <div v-else-if="childLink.text" v-html="childLink.text" class="non-link"></div>
+                    <div v-else-if="childLink.type === 'divider'" class="divider"></div>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+              <div class="buttons-block">
+              <a class="buttons-block--button" href="https://login.okta.com/" >Login</a>
+              <a class="buttons-block--button red" href="https://developer.okta.com/signup/" >Sign Up</a>
+            </div>
+          </nav>
+          <div class="top-bar-nav--controls">
+           <button v-if="topBarCollapsed" v-on:click="toggleSearch()" :class="{'search-icon': true, 'active': showSearchBar}"></button>
+           <div class="top-bar-nav--controls--toggle" @click="topBarCollapsed = !topBarCollapsed">
```

Comment:
can we use a different prop name? (e.g. burgerMenuOpened)

Reply:
Yes, it's a good one. Renamed.